src/cassandra/cli/load.cdl

```diff
@@ -0,0 +1,180 @@
+-- this cdl schema create the main tables for blueflood in cassandra using the cqlsh client
+-- usage:
+--   $ cqlsh hostname -f load.cdl -u username -p password
+-- example:
+--   $ cqlsh 127.0.0.1 -f ./blueflood/src/cassandra/cli/load.cdl
+
+CREATE KEYSPACE IF NOT EXISTS "DATA"
+    WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };
+
+
+CREATE TABLE IF NOT EXISTS "DATA".metrics_locator (
+    key bigint,
+    column1 text,
+    value text,
+    PRIMARY KEY (key, column1)
+) WITH COMPACT STORAGE AND speculative_retry = 'NONE';
```

Comment:
read about COMPACT STORAGE here:  http://blog.librato.com/posts/cassandra-compact-storage

but a quick quote/snippet from it:
```
mainly targeted towards backward compatibility for definitions created before CQL3... provides a slightly more compact layout of data on disk but at the price of diminished flexibility and extensibility... is not recommended outside of the backward compatibility reason evoked above.

Prior to CQL when the Thrift interface was the only API, all tables were built with compact storage. As the documentation states though, it is a legacy option not intended for anything besides backwards compatibility.
...
In the article, they mentioned they saw a 30x storage size overhead when not using compact storage on a table.
...
Our primary use case for Cassandra is large time-series data set storage. We must optimize for maximum on-disk efficiency since our storage load and hence costs are directionally proportional to the usage of the service. An extra 35% overhead on the storage footprint is significant.
...
If storage efficiency isn’t your primary driving factor, then the schema flexibility and transactional updates that a non-compact table provides can make application development easier.
```

Read about speculative_retry here:  http://www.datastax.com/dev/blog/rapid-read-protection-in-cassandra-2-0-2

but basically it does this:
```
Rapid read protection can be configured to do retry after a fixed period of milliseconds or after a percentile of the typical read latency (tracked per table).
```

TLDR:  both options are really good to have tables created in a local cassandra cluster for testing.  The COMPACT STORAGE option specifically should be apply to any cassandra tables created for blueflood, since it fits the bill for what we're doing (time series data with very little